paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSchema.java

```diff
@@ -36,7 +38,7 @@ public class MySqlSchema {
     private final String databaseName;
     private final String tableName;
 
-    private final LinkedHashMap<String, DataType> fields;
+    private final Map<String, Tuple2<DataType, String>> fields;
```

Comment:
Here the order of fields is important, so the definition is `LinkedHashMap` to make sure it will be initialized to an ordered map. We don't need to modify it.